Source/CesiumRuntime/Public/CesiumGlobeAnchorComponent.h

```diff
@@ -458,4 +470,9 @@ class CESIUMRUNTIME_API UCesiumGlobeAnchorComponent : public UActorComponent {
    * globe transform.
    */
   void _updateCartographicProperties();
+
+  /**
+   * Rotation offset applied to the Actor when snapping to east/south/up.
+   */
+  FRotator _rotationOffset;
```

Comment:
This should be a `UProperty` so it gets saved and restored right? Or should the offset be reset everytime like it is now? Related to that, you could make this `EditAnywhere` so that it can be set just once in the editor. What do you think?

Reply:
Yeah those adjustments could work well, it would be ok to set the offset variable directly as well within the code without the get/set as there's no additional functionality.